types/k6/encoding.d.ts

```diff
@@ -1,14 +1,30 @@
 /**
  * Base64 decode a string.
- * https://k6.io/docs/javascript-api/k6-encoding/b64decode-input-encoding
- * @param input - Base64 encoded string or ArrayBuffer object.
+ * https://k6.io/docs/javascript-api/k6-encoding/b64decode-input-encoding-format/
+ * @param input - The string to base64 decode.
  * @param encoding - Base64 variant.
  * @returns Decoded string.
```

Comment:
Hhmm why is `b64decode()` repeated? We need separate versions with and without the `format` argument?

The `@returns` description here should probably match the one below, though.

Reply:
> We need separate versions with and without the format argument

I think we need this to return the correct type depending on the format parameter. See the tests https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53101/commits/d5387b709bc015bab366bccffc496839b36da955


> The @returns description here should probably match the one below, though.

done https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53101/commits/d2653b966428c44f5f0218b782dcd0b516f9fca2